test/extended/util/framework.go

```diff
@@ -870,6 +965,20 @@ var CheckPodIsSucceededFn = func(pod kapi.Pod) bool {
 	return pod.Status.Phase == kapi.PodSucceeded
 }
 
+// CheckPodIsReadyFn returns true if the pod's ready probe determined that the pod is ready.
+var CheckPodIsReadyFn = func(pod kapi.Pod) bool {
```

Comment:
It could, I'm just following a convention used to define other callback function defined in this file. See https://github.com/openshift/origin/blob/313c88ce55f3f109e2d8c9d4dd87fcd750e16eda/test/extended/util/framework.go#L964, https://github.com/openshift/origin/blob/313c88ce55f3f109e2d8c9d4dd87fcd750e16eda/test/extended/util/framework.go#L492, https://github.com/openshift/origin/blob/313c88ce55f3f109e2d8c9d4dd87fcd750e16eda/test/extended/util/framework.go#L600 as examples. I'm not really sure why, but I'd rather stay consistent.